lib/manageiq/network_discovery/discovery.rb

```diff
@@ -1,14 +1,19 @@
 require 'net/ping'
 
+# The discovery module are loaded on demand, this is not needed for the provider probes as they are plugins
```

Comment:
You also shouldn't need this since `lib` is already on the load path.

Reply:
Thanks, done.